portlist/poller.go

```diff
@@ -62,20 +64,34 @@ type osImpl interface {
 }
 
 // newOSImpl, if non-nil, constructs a new osImpl.
-var newOSImpl func() osImpl
+var newOSImpl func(withLoopbackUpdates bool) osImpl
 
 var errUnimplemented = errors.New("portlist poller not implemented on " + runtime.GOOS)
 
+type Option func(p *Poller)
+
+// WithLoopBackUpdates tells the Poller to send updates for new ports
+// that are opened on a loopback address. The default is that loopback
+// addresses are skipped.
+func WithLoopbackUpdates(b bool) Option {
+	return func(p *Poller) {
+		p.withLoopbackUpdates = b
+	}
+}
```

Comment:
@bradfitz yeah it's def not my favorite either, but otherwise changing `NewPoller` would be a breaking change. Is it okay to make a breaking change here?

Reply:
Some options can be: 

1.
```golang
type PollerOptions struct { WithLoopback bool }
func NewPoller(opts PollerOptions) ...
```

2.
```golang
func NewPoller(withLoopback bool) ...
```

```golang
func NewPoller() ... // deprecated.
func NewPollerWithOptions(...) ... // non breaking change, but now we have two constructors.
```